corehq/apps/style/static/preview_app/less/preview_app/variables.less

```diff
@@ -0,0 +1,2 @@
+@preview-phone-width: 250px;
+@preview-tablet-width: 450px;
```

Comment:
@biyeun i redefined these here, not sure if you have any thoughts as to whether or not that makes sense or not